src/app/CommandHandler.cpp

```diff
@@ -550,6 +554,18 @@ CHIP_ERROR CommandHandler::FinishStatus()
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR CommandHandler::RollbackResponse()
+{
+    VerifyOrReturnError(mState == State::Idle || mState == State::Preparing || mState == State::AddingCommand,
```

Comment:
When would RollbackResponse happen in the Idle state?  I guess if `AllocateBuffer` fails?  But then `mBackupWriter` wouldn't be in a reasonable state anyway, so we shouldn't use it in that case.